docker/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PATH "/root/.poetry/bin:$PATH"
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
```

Comment:
Is there a (versioned) container that can be used?

Reply:
I think this is inferred in the `pyproject.toml` file.

Reply:
I'm referring to Poetry itself, like https://hub.docker.com/_/composer.

Reply:
I see. There are some floating around on Docker Hub but not an official image.

Reply:
From what we have seen yesterday, no luck: have to install Poetry on top of a `python` official image